### DISCUSSION TITLE
Upgrade local mode from v0.12.1 to v0.13.0

### DISCUSSION THREAD

**Discussion by ngiagtzogloupsw** [openobserve/openobserve#4942] [Q&A]
Hello all,

Upgrading from v0.12.1 to v0.13.0 causes the following error when trying to query logs:

Search SQL execute error
TraceID:9b153a0ce1ee42deac048600a9aaca74
Error during planning: table 'datafusion.public.streamName' not foundSearch SQL execute error
TraceID:9b153a0ce1ee42deac048600a9aaca74
Error during planning: table 'datafusion.public.streamName' not found

Volume is the same as in previous versions. previous upgrades worked fine.

Docker command:

docker run -d --name openobserve-qa-0.13.0 -v /c/work/openobserve/data:/data -p 4317:4317 -p 5080:5080 -e ZO_GRPC_PORT=4317   --restart=always public.ecr.aws/zinclabs/openobserve:v0.13.0

Tried also specifying explicitly the mode and store using -e ZO_LOCAL_MODE=true -e ZO_META_STORE=sqlite

Has anything changed that needs manual intervention?

Thank you

**Comment by hengfeiyang** [comment]
What is your query?

**Reply by ngiagtzogloupsw** [reply]
Currently using 0.12.1 without the issue I specified above which occurs when I select a stream and click Run Query in the Logs tab.

In the past I have upgraded to 0.12.1 from a previous version using the exact same process without any issues.

If I stop the 0.13.0 container where I encounter this issue and start the old 0.12.1 one the issue does not exist and I can Query data normally.

What I am asking is if there anything I might have missed that needs to be done, other than just spinning up a container for the version 0.13.0 (replacing the current 0.12.1 one) and using the same volume for data as before?

**Reply by hengfeiyang** [reply]
Nothing special, just upgrade, maybe you should refresh your browser after upgrade?

**Reply by ngiagtzogloupsw** [reply]
I just upgraded to 0.13.1-rc1 as well and still encounter the exact same issue upon using Run Query (Adding image for reference):

I tried an entirely new browser after the upgrade and the issue persists.

<img width="500" alt="image" src="https://github.com/user-attachments/assets/75304c77-d840-4068-a1d2-dde8c5725a34">

Once I revert back to 0.12.1 it works as before.

**Reply by hengfeiyang** [reply]
What is your query? What is the SQL? What is your stream name?

**Reply by ngiagtzogloupsw** [reply]
Sorry for the delay.
The query is the default one when selecting stream `QA` from the dropdown: 

In SQL mode it is:

`SELECT * FROM "QA"`

**Reply by hengfeiyang** [reply]
This is a bug with Upper stream name, Can you help create an issue? thank you.

**Reply by ngiagtzogloupsw** [reply]
Opened https://github.com/openobserve/openobserve/issues/4980

**Reply by ngiagtzogloupsw** [reply]
@hengfeiyang 

[Release v0.13.1-rc3](https://github.com/openobserve/openobserve/releases/tag/v0.13.1-rc3) Resolves the log issue.

I noticed also on v0.13.1-rc3 that traces fetch no results for the trace stream with the same name `QA` as the previous log stream.

Could this be caused by a similar issue? Using filters or none at all makes no difference. v0.12.1 fetches traces correctly.

**Reply by hengfeiyang** [reply]
I think the same issue, by the way with `v0.13.1-rc3` we no longer support Upper stream, everything need to be lowercase.

**Reply by hengfeiyang** [reply]
You can comment under the issue, to mentioned the same issue in traces, we didn't test for traces. we only have the `default` stream for traces.😁

**Reply by ngiagtzogloupsw** [reply]
Good to know. Is there a way to rename any existing streams without repercussions?

**Reply by hengfeiyang** [reply]
not yet, we can't rename stream.
